worker_option.go

```diff
@@ -31,6 +31,13 @@ func WithWorkerQueue(queue string) WorkerOption {
 	}
 }
 
+// WithWorkerJobTypes limits/filters the job types this worker will fetch from the DB.
+func WithWorkerJobTypes(jobTypes []string) WorkerOption {
```

Comment:
should be variadic to keep the options consistent

Reply:
Will change! :+1: